app/models/goal.py

```diff
@@ -1,6 +1,14 @@
 from flask import current_app
 from app import db
-
+from app.models.task import Task
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", backref="task", lazy=True)
```

Comment:
The `backref` argument is used to provide a name that SqlAlchemy will use to add a property to the target model (`Task` in this case) to get back to the current model `Goal`. `goal` would be an appropriate name.

Then, if we have a `Task` instance, we could write `task.goal` to get a reference to the Goal with which the Task is associated (or `None` if the Task isn't associated with a Goal).

The `tasks` association itself would let us write `goal.tasks` to get the collection of all the Tasks associated with a particular Goal.

```suggestion
    tasks = db.relationship("Task", backref="goal", lazy=True)
```